versioned_sidebars/version-2.1-sidebars.json

```diff
@@ -278,8 +278,16 @@
                     "label": "Analyzing Database",
                     "items": [
                         "lakehouse/database/es",
-                        "lakehouse/database/jdbc",
-                        "lakehouse/database/max-compute"
+                        "lakehouse/database/max-compute",
+                        "lakehouse/database/jdbc-overview",
```

Comment:
Put `jdbc-overview` at first.
Follow with other jdbc doc, and then es and max compute